test/format/format_config_def.c

```diff
@@ -279,6 +279,11 @@ CONFIG configuration_list[] = {{"assert.read_timestamp", "assert read_timestamp"
 
   {"ops.salvage", "configure salvage", C_BOOL, 100, 1, 0, V_GLOBAL_OPS_SALVAGE},
 
+  {"ops.throttle", "enable delay between ops", C_BOOL, 10, 0, 0, V_GLOBAL_OPS_THROTTLE},
+
+  {"ops.throttle.sleep_us", "average duration of sleep between ops per table, us", 0x0, 0, M(1),
```

Comment:
Discussion: Would it make more sense to define this as `operation throughput per table` rather than the delay between ops? The delay seems like the mechanism to achieve throttling, and throughput the end goal.

Reply:
We considered using parameters like operations/sec or bytes/sec. Unfortunately, viability of this kind of targets highly depends on many other parameters and machine's capabilities. It'd be hard to even tell a reasonable range for such parameter. A bad choice of parameter range would lead to insufficient coverage of cases on the one hand and wasted runs on the other hand where the target is not achievable.

Meanwhile, the intent behind this change is to give a chance to background threads to work. This is easily achievable with a simple delay between operations.

It was decided to start with this simple implementation and see if we need to extend it later.